tests.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+
+SITE_LOCATION = "http://localhost:5000"
+
+
+def url_ok(url):
+    try:
+        r = urllib.request.urlopen(url).getcode()
+
+    except:
+        r = 404
+
+    return r
+
+print(url_ok(SITE_LOCATION))
```

Comment:
You don't need to use `print` in a test file, pytest typically takes case of managing the collection and display of test results for you.

Reply:
print was removed and I followed the correct use of pytest.